python/lsst/afw/fits/fits.cc

```diff
@@ -57,16 +221,28 @@ PYBIND11_PLUGIN(_fits) {
     clsMemFileManager.def("setData", [](MemFileManager &m, py::bytes const &d, size_t size) {
         memcpy(m.getData(), PyBytes_AsString(d.ptr()), size);
     });
-
-    /* Module level */
-
-    /* Member types and enums */
-
-    /* Constructors */
-
-    /* Operators */
-
-    /* Members */
+    clsMemFileManager.def("readMetadata",
+                          [](MemFileManager & self, int hdu=INT_MIN, bool strip=false) {
+                              return readMetadata(self, hdu, strip);
+                          }, "hdu"_a=INT_MIN, "strip"_a=false);
+
+    defineImageCompression(mod);
+    defineImageScalingOptions(mod);
+    defineImageScale(mod);
+    defineImageWriteOptions(mod);
+    defineFits(mod);
+
+    mod.def("readMetadata",
+            [](std::string const& filename, int hdu=INT_MIN, bool strip=false) {
+                return readMetadata(filename, hdu, strip);
+            }, "fileName"_a, "hdu"_a=INT_MIN, "strip"_a=false);
+    mod.def("setAllowImageCompression", &setAllowImageCompression, "allow"_a);
+    mod.def("getAllowImageCompression", &getAllowImageCompression);
```

Comment:
I'm not crazy about globals either, but I thought it might be a good idea to allow the user to nuke the whole idea of compression, since the goal has been to have compression used by default --- they wouldn't have to discover exactly what obscure knob to twiddle to deactivate it, but just call this function.